Blogpost: XSF sign open letter to EC (#1422)
xsf/xmpp.org: content/blog/2024-08-10-XSF_signs_open_letter_european_commission.md

---
title: The XSF signs Open Letter to the European Commission
date: 2024-01-05
author: XMPP Standards Foundation

> categories: ["XSF Organisational", "FOSDEM"]

Reviewer: ```suggestion
categories: ["XSF Organisational"]
```

---

As currently many other organisations doing, the XMPP Standards Foundation (XSF) has decided to also sign the Open Letter to the European Commission initiated by [Petites Singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/) and translated to English by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter)

# The European Union must keep funding free software

> Initially published by [petites singularités](https://ps.zoethical.org/pub/lettre-publique-aux-ncp-au-sujet-de-ngi/). English translation provided by [OW2](https://www.ow2.org/view/Events/The_European_Union_must_keep_funding_free_software_open_letter). 

Reviewer: Repetition of above

> If you want to sign the letter, please publish the letter on your website and complete the [table below](https://pad.public.cat/lettre-NCP-NGI).

Reviewer: There is no "table below"

Author: Thanks, but I will ignore it as it is not from us.

This is all the more essential in the current geopolitical context: the challenge of technological sovereignty is central, and free software allows to address it while acting for peace and sovereignty in the digital world as a whole.

In this perspective, we urge you to claim for preserving the NGI programme as part of the 2025 funding programme.

[^1]: As defined by Horizon Europe, widening Member States are Bulgaria, Croatia, Cyprus, the Czech Republic, Estonia, Greece, Hungary, Latvia, Lituania, Malta, Poland, Portugal, Romania, Slovakia and Slovenia. Widening associated countries (under condition of an association agreement) include Albania, Armenia, Bosnia, Feroe Islands, Georgia, Kosovo, Moldavia, Montenegro, Morocco, North Macedonia, Serbia, Tunisia, Turkey and Ukraine. Widening overseas regions are : Guadeloupe, French Guyana, Martinique, Reunion Island, Mayotte, Saint-Martin, The Azores, Madeira, the Canary Islands.

## Statement by the XMPP Standards Foundation

> The XMPP Standards Foundation supports the initiative as well as the statements made. In the past, many [XMPP projects](https://ngi.eu/?s=xmpp) have been funded by the European Union to enhance realtime infrastructe with our standard but also drive innovation we as Community are building for more than two decades and is well established in the European digital landscape.

Reviewer: ```suggestion
The XMPP Standards Foundation supports this initiative as well as the statements made.
In the past, many [XMPP projects](https://ngi.eu/?s=xmpp) have been funded by NGI programmes to enhance realtime infrastructure using established XMPP standards.
These funds significantly helped drive innovation in a community with two decades of history in the European digital landscape.
```

> Below name some of the current project that also show project that go beyond user applications and are also part of EU strategic initiatives:

Reviewer: ```suggestion
These are some of the funded projects:
```
